### PR TITLE
Use service well-known name

### PR DESCRIPTION
**Pull request by joseivanlopez**
According to documentation, it is recomended to use the so-called well-known name for the dbus service and dbus policy, see https://dbus.freedesktop.org/doc/dbus-api-design.html#service-files. Moreover, the dbus service should be distinguished from the systemd service, see https://nyirog.medium.com/register-dbus-service-f923dfca9f1. 

This PR also includes small reoganization of some files.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -12,9 +12,9 @@ sudo zypper --non-interactive in --no-recommends \
   cockpit
 
 sudo systemctl start cockpit
-sudo systemctl start d-installer
+sudo systemctl start org.opensuse.DInstaller
```
Comment (jreidinger) [inline deploy.sh:15]: this looks strange, as I am not aware about any other systemd service that use such name.

File: README.md [hunk 2]
```diff
@@ -63,25 +63,25 @@ service using the D-Bus interface it provides.
...
-    busctl call org.opensuse.YaST /org/opensuse/YaST/Installer \
-      org.opensuse.YaST.Installer GetDisks
+    busctl call org.opensuse.DInstaller /org/opensuse/DInstaller/Language \
+      org.opensuse.DInstaller.Language AvailableLanguages
 
 If you want to get the properties, just type:
 
-    busctl call org.opensuse.YaST /org/opensuse/YaST/Installer \
-      org.freedesktop.DBus.Properties GetAll s org.opensuse.YaST.Installer
+    busctl call org.opensuse.DInstaller /org/opensuse/DInstaller/Language \
```
Comment (imobachgs) [inline README.md:83]: I would say the correct paths are `/org/opensuse/DInstaller/Language1` and `org.opensuse.DInstaller.Language1`. At least, that's how the code looks like now.

File: README.md [hunk 3]
```diff
@@ -63,25 +63,25 @@ service using the D-Bus interface it provides.
 
 Beware that `d-installer` must run as root (like YaST does) to do hardware probing, partition the
 disks, install the software and so on. So you need to tell dbus about the service by copying
-`yastd/share/dbus.conf` to `/etc/dbus-1/system.d/d-installer.conf`.
+`service/config/d-installer.conf` to `/etc/dbus-1/system.d/org.opensuse.DInstaller.conf`.
```
Comment (imobachgs) [inline README.md:66]: I like changing the destination, but I am not so sure about the original file.

**Review by imobachgs — COMMENTED** [review]
In general, I am not sure about changing the original paths.

File: setup.sh [hunk 4]
```diff
@@ -9,12 +9,13 @@ sudo zypper --non-interactive install gcc gcc-c++ make openssl-devel ruby-devel
 sudo systemctl start cockpit
 
 # set up the d-installer service
-sudo cp yastd/share/dbus.conf /etc/dbus-1/system.d/d-installer.conf
-cd yastd; bundle config set --local path 'vendor/bundle'; bundle install; cd -
+sudo cp service/config/d-installer.conf /etc/dbus-1/system.d/org.opensuse.DInstaller.conf
```
[the remark this answers is not in the document]
Comment (imobachgs) [reply setup.sh:12]: Or a polkit configuration.

File: service/d-installer.gemspec [hunk 5]
```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |spec|
   spec.email = "yast-devel@opensuse.org"
   spec.homepage = "https://github.com/yast/d-installer"
   spec.license = "GPL-2.0-only"
-  spec.files = Dir["lib/**/*.rb", "bin/*", "share/*", "[A-Z]*"]
+  spec.files = Dir["lib/**/*.rb", "bin/*", "config/*", "[A-Z]*"]
```
[the remark this answers is not in the document]
Comment (joseivanlopez) [reply service/d-installer.gemspec:32]: Actually there is no super good reason. I simply see that *.service* file is deployed to a share dir (/usr/share/dbus-1/system-services/) but *.conf* is not (/etc/dbus-1/system.d/). So, having both under *share/* looked strange to me. And I understood that both files are somehow configurring our service: one file for indicating the bus, script, etc, and the other file to indicate the policy config.

But as said, no super reasons, so I can revert it.

File: setup.sh (hunk 4 above)
[the remark this answers is not in the document]
Comment (joseivanlopez) [reply setup.sh:12]: Umm, I am not sure if I got it: *.conf* are policy files and *.service* are service files. I get it as the polilcy and service definition for *d-installer*.

[the remark this answers is not in the document]
Comment (imobachgs) [reply setup.sh:12]: Yes, but `d-installer.conf` looks generic enough (from the name, I am not able to figure out that it is a policy). It looks like a `d-installer` configuration file.

File: service/d-installer.gemspec (hunk 5 above)
[the remark this answers is not in the document]
Comment (imobachgs) [reply service/d-installer.gemspec:32]: I can leave with that, I just wanted to know the reason. Usually, people use `share` for configuration examples, for instance. It is "independent architecture data", basically. But let's keep it as it is.

File: setup.sh (hunk 4 above)
[the remark this answers is not in the document]
Comment (imobachgs) [reply setup.sh:12]: Actually, what I said is not true, they use `/usr/share/doc` instead.

File: service/d-installer.gemspec (hunk 5 above)
[the remark this answers is not in the document]
Comment (jreidinger) [reply service/d-installer.gemspec:32]: @joseivanlopez well, as we have system service from vendor we should in fact use `/usr/share/dbus-1/system.d/`

[the remark this answers is not in the document]
Comment (imobachgs) [reply service/d-installer.gemspec:32]: Actually, what I said is not true, they use `/usr/share/doc` instead.

**Review by jreidinger — APPROVED** [review]

**Review by imobachgs — APPROVED** [review]
Thanks a lot for your patience! :-)

**Pull request merged**